src/service/extension-location.js

```diff
@@ -49,13 +49,19 @@ export function calculateExtensionScriptUrl(location, extensionId, isLocalDev) {
  * @param {string} entryPoint
  * @param {boolean=} isLocalDev
  * @param {boolean=} opt_rtv
+ * @param {boolean=} opt_minified
  * @return {string}
  */
 export function calculateEntryPointScriptUrl(
-    location, entryPoint, isLocalDev, opt_rtv) {
+    location, entryPoint, isLocalDev, opt_rtv, opt_minified) {
   const base = calculateScriptBaseUrl(location, isLocalDev);
-  if (opt_rtv) {
+  if (opt_rtv && opt_minified) {
```

Comment:
Not a big deal, but I think having possibly empty strings in the format string makes it harder to verify that slashes, etc. are correct. Another option is to incrementally build the url string.